### ISSUE TITLE
aria-disabled should be focusable

### ISSUE THREAD

**Issue by kee-oth** [andreasbm/focus-trap#13]
Currently, `focus-trap` doesn't allow for focusing of elements with `aria-disabled=true`. For accessibility reasons, `aria-disabled=true` is often used instead of `disabled` to allow for the element to be tabbed to. Can we at least get an opt-in option to not consider elements with `aria-disabled=true` when determining whether it can be focused?

Thanks!

https://github.com/andreasbm/focus-trap/blob/efe549115288362d2aaae9b7a4780cc4539d097d/src/lib/focusable.ts#L29
